src/webview/create-component/createComponentLoader.ts

```diff
@@ -92,6 +92,12 @@ export default class CreateComponentLoader {
         );
         panel.webview.html = await loadWebviewHtml('createComponentViewer', panel);
         CreateComponentLoader.panel = panel;
+
+        if (folderPath) {
```

Comment:
There are a few options to make this code simpler:
- If the `folderPath` isn't set, don't send the message
- Send the message regardless of the value of `folderPath`. Instead of using an if statement, let the webview determine if it's necessary to change views based on if `folderPath` contains a path